examples/volumes/vsphere/README.md

```diff
@@ -353,6 +353,200 @@
       pvpod       1/1     Running   0          48m        
       ```
 
+### Virtual SAN policy support inside Kubernetes
+####About Virtual SAN Storage Policies####
```

Comment:
Since we had no other place where we put the complete documentation for VSAN policy support in Kubernetes. I went ahead with full description in this place.